libcodechecker/storage/report_server.py

```diff
@@ -381,8 +381,7 @@ def storeReportInfo(self,
             path_start = path_ids[0].id if len(path_ids) > 0 else None
 
             source_file = self.session.query(File).get(file_id)
-            source_file_path, source_file_name = ntpath.split(
-                source_file.filepath)
+            _, source_file_name = ntpath.split(source_file.filepath)
```

Comment:
If we are already touching this line, I would like to see `ntpath` gone. `ntpath` is `os.path` on a Windows system, while we are not even targeting Windows as of now.

`os.path` is always the appropriate for the current execution environment, there isn't really a need to specifically use `ntpath` or `posixpath` at places.